src/app/(auth)/recoverpassword/page.tsx

```diff
@@ -0,0 +1,21 @@
+// meta tags
+import type { Metadata } from 'next';
+export const metadata: Metadata = {
+	title: 'Recover Password Page',
+	description: 'Join us at CAMEL.',
```

Comment:
Change description for this metadata description. Not necessarily required just should align with page if you include such pages..

Same idea follows for both the /recoverpassword route and the /updatepassword route pages. 

"**CAMEL - Password Reset Page**" or "**CAMEL - Securely Reset Your Password**" or something

Reply:
Descriptions are being taken care of to fit pages.